### ISSUE TITLE
Enter insert mode automatically on certain sites

### ISSUE THREAD

**Issue by collinpeters** [1995eaton/chromium-vim#202]
I am trying with certain sites that support keyboard nav like Gmail and Jira, to automatically enter insert mode. I don't want to outright blacklist the sites. I have tried the following but it is not working:
```
site 'https://intouch.atlassian.net/*' {
      call insertMode
}
```

**Comment by 1995eaton** [comment]
See if it's working now

**Comment by collinpeters** [comment]
Works when I installed the master.zip! Very cool. When would this be officially released?

**Comment by 1995eaton** [comment]
I plan on pushing an update to the store in the next few days
